codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/CodegenVisitor.java

```diff
@@ -327,4 +328,12 @@ public Void serviceShape(ServiceShape shape) {
         });
         return null;
     }
+
+    @Override
+    public Void intEnumShape(IntEnumShape shape) {
+        if (!shape.getEnumValues().isEmpty()) {
```

Comment:
This condition is always `true`, we do validate that enums have at least one member.